source/Analysis/include/AnalyzerGeometry.h

```diff
@@ -37,6 +37,16 @@ class TProfile;
 class Ring;
 class RootWTable;
 
+
+// Formatting parameters
+static const int coordPrecision = 3;
+static const int zOverlapPrecision = 5;
+static const int anglePrecision = 1;
+// Some strings for the html formatting
+static const std::string subStart   = "<sub>";      // These only should be needed
+static const std::string subEnd     = "</sub>";
+
```

Comment:
Hi Zbynek, yes sure !
Sorry this is just 'first rush' 3000 code lines to make things working :)
The code here is not bound nor even intended at all to be with style, but to provide consistent results :)

Maybe let's look at the geo results themselves at runtime if there are osies, then the style can be improved later (and on dev_gabie as well ;) )